app/controllers/patients_controller.rb

```diff
@@ -38,7 +38,7 @@ def create
 
   def update
     respond_to do |format|
-      if @patient.update(patient_with_parsed_medical_history)
+      if @patient.update(patient_with_parsed_medical_history.merge(author: current_user))
```

Comment:
Since the data snapshot was bad, wherein several Patients didn't have an author, we thought that may have been the case in production also. It makes sense that an edit should not override the original author in reality (also given the fact that it must always have an author in reality).

Will remove this bit.